inlong-manager/manager-plugins/src/main/assembly/package.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we reuse the assembly.xml in manager-web？

Reply:
reuse same assembly.xml ,manager-plugins can not package independently